partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnEq.kt

```diff
@@ -25,12 +26,13 @@ import org.partiql.types.PType
 internal object FnEq : Function {
 
     // Memoize shared variables
+    private const val NAME = FunctionUtils.SYSTEM_PREFIX_INTERNAL + "eq"
```

Comment:
Rather than "computing" the name in every function definition, make it a helper.

```
val name = FunctionUtils.hide("eq")
```